ocs_ci/ocs/constants.py

```diff
@@ -158,6 +159,11 @@
 PGSQL_STATEFULSET_YAML = os.path.join(
     TEMPLATE_PGSQL_SERVER_DIR, "StatefulSet.yaml"
 )
+
+PGSQL_BENCHMARK_YAML = os.path.join(
+    TEMPLATE_PGSQL_DIR, "Benchmark.yaml"
```

Comment:
can we rename the yaml file to pgsql_benchmark.yaml ?

Reply:
Fixed. Renamed file to PGSQL_Benchmark.yaml.